test/regress/cli/regress0/ho/model-lam-simple.smt2

```diff
@@ -0,0 +1,14 @@
+; COMMAND-LINE: -q
+; EXPECT: sat
+; EXPECT: (
+; EXPECT: (define-fun f ((x Int)) Bool (>= x 1))
```

Comment:
Why do you expect the first disjunct `(= f (lambda ((x Int)) (> x a)))` to hold and not the second one 
`(= f (lambda ((x Int)) (< x a)))`

Reply:
I can make the test more robust.